Packages/ChatSecure/ChatSecure/Classes/Networking/Services/Messsage/MessageService.swift

```diff
@@ -52,9 +56,16 @@ public class MessageService {
 		self.signalStore = signalStore
 		self.senderStore = senderStore
 	}
+	
+	public var currentRoomId: Int64 = 0
 }
 
 extension MessageService: IMessageService {
+	
+	public func updateCurrentRoom(roomId: Int64) {
+		self.currentRoomId = roomId
+	}
+	
 	public func sendMessageInPeer(senderId: String, ownerDomain: String, receiverId: String, receiverDomain: String, groupId: Int64, plainMessage: String, isForceProcessKey: Bool, cachedMessageId: Int) async -> Result<RealmMessage, Error> {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/0b09249675dff5b0a460b69f27e9f3e1d518728a/Packages/ChatSecure/ChatSecure/Classes/Networking/Services/Messsage/MessageService.swift#L69"></span>Function body should span 50 lines or less excluding comments and whitespace: currently spans 51 lines<br />
<code>function_body_length</code> <code>MessageService.swift:69</code></td>
    </tr>
  </tbody>
</table>